package ExceptionHandling2;

public class Launch7 {

	
	Launch7() throws ArithmeticException{
		throw new ArithmeticException("");
	}
	
	static void disp() {
		
	}
	public static void main(String[] args) {
		// TODO Auto-generated method stub
    //  Launch7 l = new Launch7() throws new  ArithmeticException();
		
//		try {
//		disp() throw new ArithmeticException();
//		}catch(Exception e) {
//			
//		} --> not at all possible
	}
	

}
